plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/style.scss

```diff
@@ -159,3 +159,8 @@
 		}
 	}
 }
+
+// We don't want zero specificity for this style.
+.wc-block-product-filters .hidden {
```

Comment:
I'd consider a bit more specific class name than `hidden`, e.g. `.wc-block-product-filter--hidden`.

`hidden` I think there's a high chance some 3rd-party code with non scoped selectors (simply targeting `.hidden`) could use this class to actually hide all filters.